core/common/src/main/java/alluxio/util/webui/UIMountPointInfo.java

```diff
@@ -28,6 +28,7 @@
   private long mUfsUsedBytes;
   private boolean mReadOnly;
   private boolean mShared;
+  private String mProperties;
```

Comment:
mark `final` whenever possible.
```suggestion
  private final String mProperties;
```